cmd/agent/app/options/options.go

```diff
@@ -181,6 +183,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet, allControllers []string) {
 	fs.StringVar(&o.KarmadaKubeConfig, "karmada-kubeconfig", o.KarmadaKubeConfig, "Path to karmada control plane kubeconfig file.")
 	fs.StringVar(&o.KarmadaContext, "karmada-context", "", "Name of the cluster context in karmada control plane kubeconfig file.")
 	fs.StringVar(&o.ClusterName, "cluster-name", o.ClusterName, "Name of member cluster that the agent serves for.")
+	fs.StringArrayVar(&o.ClusterLabels, "cluster-labels", []string{}, "Specify labels for the member cluster in the format <key>=<value> e.g., --cluster-labels=foo=bar. You can use this flag multiple times. Note that it reconciles the labels of the cluster object when the agent restarts.")
```

Comment:
```suggestion
	fs.StringArrayVar(&o.ClusterLabels, "cluster-labels", []string{}, "Specify labels for the member cluster in the format <key>=<value> e.g., --cluster-labels=foo1=bar1,foo2=bar2. Note that it reconciles the labels of the cluster object when the agent restarts.")
```
Does each key-value pairs separated by comma(`,`)?

Reply:
Only 1 key-value pair in each flag. E.g. 
```
        - --cluster-labels=foo1=bar1
        - --cluster-labels=foo2=bar2
```

Reply:
I'd prefer to separate them by a comma, like the `--feature-gates`. What do you think?